aptos-move/framework/aptos-framework/doc/staking_contract.md

```diff
@@ -2430,118 +2430,4 @@ The guid_creation_num of the ccount resource is up to MAX_U64.
 </code></pre>
 
 
-The Store exists under the staker.
```

Comment:
Why is this all deleted?

Reply:
It seems to be a bug when generating the doc. After removing the comment added to [staking_contract.spec.move](https://github.com/aptos-labs/aptos-core/pull/7634/files/0c1e998fa527c4edaa80b5e2e80c148bd56b4b25#diff-e9a5d468be1897db0ac84561d1bd4770f4a6a789fc18e59b18218c783057c676), this part is added back